### ISSUE TITLE
Test_openjdk16_j9_extended.openjdk_x86-64_windows: java/io/File/GetXSpace.java: InvalidPathException: Illegal char <:> at index 0: :

### ISSUE THREAD

**Issue by andrew-m-leonard** [adoptium/aqa-tests#2461]
https://trss.adoptopenjdk.net/resultSummary?parentId=606acc3047db975731c5dd89
https://ci.adoptopenjdk.net/job/Test_openjdk16_j9_extended.openjdk_x86-64_windows/17/consoleFull
```
15:59:05  java.nio.file.InvalidPathException: Illegal char <:> at index 0: :
15:59:05  	at java.base/sun.nio.fs.WindowsPathParser.normalize(WindowsPathParser.java:182)
15:59:05  	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:153)
15:59:05  	at java.base/sun.nio.fs.WindowsPathParser.parse(WindowsPathParser.java:77)
15:59:05  	at java.base/sun.nio.fs.WindowsPath.parse(WindowsPath.java:92)
15:59:05  	at java.base/sun.nio.fs.WindowsFileSystem.getPath(WindowsFileSystem.java:230)
15:59:05  	at java.base/java.io.File.toPath(File.java:2316)
15:59:05  	at GetXSpace.compare(GetXSpace.java:219)
15:59:05  	at GetXSpace.testDF(GetXSpace.java:394)
15:59:05  	at GetXSpace.main(GetXSpace.java:428)
15:59:05  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
15:59:05  	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
15:59:05  	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
15:59:05  	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
15:59:05  	at com.sun.javatest.regtest.agent.MainActionHelper$AgentVMRunnable.run(MainActionHelper.java:298)
15:59:05  	at java.base/java.lang.Thread.run(Thread.java:853)
```
Looks like a classic Windows path issue with testcase?

**Comment by lumpfish** [comment]
This failure is the same as that described here: https://bugs.openjdk.java.net/browse/JDK-8251466.

I have tested locally that the updated regex proposed there enables the test to pass.

I suggest we exclude the test until the test case is fixed upstream.

**Comment by sophia-guo** [comment]
Closed  by #2561
